packages/teleport/src/console/components/DocumentSsh/ActionBar/ActionBar.jsx

```diff
@@ -27,7 +27,7 @@ export default function ActionBar({
 }) {
   const isScpDisabled = isDownloadOpen || isUploadOpen || !isConnected;
   return (
-    <Flex flex="0 0" alignItems="center" height="32px">
+    <Flex alignItems="center">
```

Comment:
It intent was to have this bar always with fixed dimensions, so its height or width does not depend in rendered inner content.  This is to make sure that the size of terminal container does not change when we calculate its height to create a session. Otherwise, if action bar height changes, the scrollbars will appear in the terminal. 

In other words, the size of the action bar should be always fixed even when it's empty. Will it be still the case?

Reply:
I see. No it doesn't. I added back the flexing to prevent growth/shrinkage and height (since the button was height 24px, I went with the same value)